payment-methods.adoc

```diff
@@ -0,0 +1,285 @@
+.How were the first payment accounts signed if there were no peers to sign them?
+====
+Bisq arbitrators signed a collection of accounts that met certain criteria to bootstrap the network of trusted payment accounts just before the v1.2 release.
+
+The following criteria were determined to balance high integrity with volume (so that there were a sufficient number of signed peer accounts available to sign other peer accounts in the beginning):
+
+* payment account must be older than 60 days
+* payment account must have been involved in a dispute as a bitcoin buyer, and received bitcoin as an outcome of that dispute (thereby proving that a successful fiat _payment_ took place)
+
+All payment accounts that met the 2 criteria above were signed by arbitrators and immediately able to sign other payment accounts upon the release of v1.2.
+====
+
+2. **Successfully settle the trade.** When you send your fiat payment to the seller, your payment account will be signed when the seller clicks the `Confirm payment received` button.
++
+Thirty days after signing, the following will take place:
++
+--
+* the 0.01 BTC trading limit will be lifted
+* trading limits will apply as if account aging began at the time of signing
+* you will be able to sign other payment accounts
+--
++
+NOTE: The purpose of the 30-day delay is to allow time for chargebacks, as the typical trade period is often not long enough for fraudulent charges to be detected. This is why you should send payment as quickly as possible when looking to get signed, and why sellers will take as long as possible (within the trade period) to confirm your payment on their end.
++
+To clarify how trading limits apply, let's look at a concrete example with SEPA. SEPA payments have a maximum trade size of 0.25 BTC, which phases in over 3 months. The limit is 0.0625 BTC in the first month, 0.125 BTC in the second month, and 0.25 BTC afterward. Let's assume we create a SEPA payment account on 01 January 2020. Then let's assume that the account is not signed until 01 March 2020. Even though the account is 60 days old at the time of signing, it will retain its 0.01 BTC trade limit through January, February, _and_ March. Then, on 01 April 2020 (30 days after signing), the payment account will drop its 0.01 BTC limit and take on a 0.125 BTC trade limit. Why? Because at that point, it's over 30 days old _as of the date it was signed_.
+
+3. **Sign other users.** No one likes 0.01 BTC limits, and many users will want to be signed by a peer with a trusted account. Once you're signed, please try to maintain some offers to give unsigned buyers a chance to be signed to broaden the trusted network.
+
+=== Things to Note
+
+* Selling limits are unaffected by account signing, and instead follow account aging. This means you can create a payment account to sell bitcoin using a risky payment method and sell 0.0625 BTC right away. You will only be able to _buy_ 0.01 BTC with that account, however, until it is signed.
```

Comment:
Oh, the sell limit is mentioned here, perhaps still worth mentioning earlier for those that are too lazy to read all of it